luminoth/utils/config.py

```diff
@@ -1,24 +1,28 @@
-import easydict
 import os.path
 import tensorflow as tf
 import yaml
 
+from easydict import EasyDict
 
-def load_config(filename):
-    return easydict.EasyDict(yaml.load(tf.gfile.GFile(filename)))
+
+def load_config(filenames, overwrite=False, warn_overwrite=False):
```

Comment:
Do we want to offer the `overwrite` option here? Setting to `False` (default) will read the config files right-to-left instead of left-to-right. If that's the desired behavior, maybe the caller should be required to reverse the list (making it easier to reason about).

Reply:
I agree. I don't really see a case where `overwrite=False` would be useful.